<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectSettingsDialog</class>
 <widget class="QDialog" name="ProjectSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QFrame" name="frame">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>551</width>
         <height>411</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Project name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="projectNameLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Designer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="designerLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Comment</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="commentLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>File created by</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="fileCreatedByLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Units</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="unitsComboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="shadeUnderwaterCheckBox">
          <property name="text">
           <string>Shade underwater</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="savePreviewImageCheckBox">
          <property name="text">
           <string>Save preview image</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="simplifyIntersectionsCheckBox">
          <property name="text">
           <string>Simplify intersections</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="mainDimensionsTab">
      <attribute name="title">
       <string>Main dimensions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Length</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lengthLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="lengthUnitLabel">
           <property name="text">
            <string>[ft]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Beam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="beamLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="beamUnitLabel">
           <property name="text">
            <string>[ft]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Draft</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="draftLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="draftUnitLabel">
           <property name="text">
            <string>[ft]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Midship location</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="midshipLocationLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="midshipLocationUnitLabel">
           <property name="text">
            <string>[ft]</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="defaultMidshipLocationCheckBox">
           <property name="text">
            <string>Default at 0.5 * L</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="hydrostaticsTab">
      <attribute name="title">
       <string>Hydrostatics</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Water density</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="waterDensityLineEdit"/>
         </item>
         <item>
          <widget class="QLabel" name="waterDensityUnitLabel">
           <property name="text">
            <string>[lbs/ft3]</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Appendage coefficient</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="appendageCoefficientLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Coefficients based on</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="coefficientComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="disableSurfaceCheckCheckBox">
         <property name="text">
          <string>Disable surface check</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Display hydrostatic features</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Select which of the following hydrostatic properties you want to have displayed in the model:</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="displacementCheckBox">
            <property name="text">
             <string>Displacement and center of buoyancy</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="sectionalAreasCheckBox">
            <property name="text">
             <string>Curve of sectional areas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="metacentricCheckBox">
            <property name="text">
             <string>Metacentric height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="longitudinalCheckBox">
            <property name="text">
             <string>Longitudinal center of floation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="lateralAreaCheckBox">
            <property name="text">
             <string>Lateral area and center of effort</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ProjectSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ProjectSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
